zulipterminal/cli/run.py

```diff
@@ -181,11 +185,13 @@ def main(options: Optional[List[str]]=None) -> None:
     except ServerConnectionFailure as e:
         print(in_color('red',
                        "\nError connecting to Zulip server: {}.".format(e)))
+        logging.exception(e)
```

Comment:
We've handled the exception here - why do we need to log it?

Reply:
I think a full traceback of even a handled exception could be useful. right?